src/helpers/getPDFCoverPage.ts

```diff
@@ -0,0 +1,65 @@
637 0-3.0557 0.4768-3.876 1.4304-0.82032 0.95362-1.2305 2.5276-1.2305 4.7219v8.844h-2.5532v-16.857h2.0764l0.41528 2.3071h0.12305c0.52295-0.83057 1.2561-1.474 2.1995-1.9303s1.9944-0.68445 3.1531-0.68445c2.0303 0 3.5581 0.48962 4.5835 1.4689s1.5381 2.5455 1.5381 4.6989v10.997h-2.5532zm13.243-1.7996c0.45118 0 0.88696-0.033324 1.3074-0.099975 0.42041-0.066651 0.75366-0.13586 0.99976-0.20764v1.9534c-0.27686 0.1333-0.68444 0.24353-1.2228 0.33069s-1.0228 0.13074-1.4535 0.13074c-3.2608 0-4.8911-1.7175-4.8911-5.1526v-10.028h-2.4148v-1.2305l2.4148-1.0613 1.0767-3.5991h1.4766v3.9067h4.8911v1.9841h-4.8911v9.9207c0 1.0151 0.24096 1.7944 0.7229 2.3379s1.1433 0.81518 1.9841 0.81518zm8.2288 1.7996h-2.5532v-16.857h2.5532v16.857zm-2.7686-21.426c0-0.58448 0.14355-1.0126 0.43066-1.2843 0.28711-0.27173 0.64599-0.40759 1.0767-0.40759 0.41016 0 0.76391 0.13843 1.0613 0.41528 0.29736 0.27686 0.44604 0.70239 0.44604 1.2766s-0.14868 1.0023-0.44604 1.2843c-0.29736 0.28198-0.65112 0.42297-1.0613 0.42297-0.43067 0-0.78955-0.14099-1.0767-0.42297-0.28711-0.28198-0.43066-0.71008-0.43066-1.2843zm18.549 21.426l-0.50757-2.3994h-0.12305c-0.84082 1.0562-1.6791 1.7714-2.5148 2.1456-0.8357 0.37427-1.879 0.5614-3.13 0.5614-1.6714 0-2.9813-0.43066-3.9298-1.292s-1.4227-2.0867-1.4227-3.676c0-3.4043 2.7224-5.1885 8.1672-5.3525l2.8608-0.092286v-1.0459c0-1.3228-0.28454-2.2994-0.85364-2.9301-0.5691-0.63062-1.4791-0.94592-2.7301-0.94592-1.4048 0-2.9941 0.43066-4.7681 1.292l-0.78442-1.9534c0.83057-0.45117 1.7406-0.80493 2.7301-1.0613 0.98951-0.25635 1.9816-0.38452 2.9762-0.38452 2.0098 0 3.4991 0.44604 4.4681 1.3381 0.969 0.8921 1.4535 2.3225 1.4535 4.2913v11.505h-1.8918zm-5.7678-1.7996c1.5894 0 2.8378-0.43579 3.7452-1.3074 0.90748-0.87159 1.3612-2.0918 1.3612-3.6606v-1.5227l-2.5532 0.10767c-2.0303 0.071778-3.494 0.38708-4.3912 0.94592-0.89722 0.55884-1.3458 1.4278-1.3458 2.6071 0 0.92286 0.27942 1.6252 0.83826 2.1072 0.55884 0.48194 1.3407 0.7229 2.3456 0.7229zm15.473 1.7996h-2.5532v-23.933h2.5532v23.933zm16.288-4.5989c0 1.5689-0.58447 2.7788-1.7534 3.6299-1.169 0.85108-2.8096 1.2766-4.9219 1.2766-2.2354 0-3.9785-0.35376-5.2295-1.0613v-2.3687c0.81006 0.41016 1.6791 0.73315 2.6071 0.96899s1.8226 0.35376 2.684 0.35376c1.333 0 2.3584-0.21277 3.0762-0.6383s1.0767-1.0741 1.0767-1.9457c0-0.65625-0.28454-1.2176-0.85364-1.6842s-1.6791-1.0177-3.33-1.6534c-1.5689-0.58448-2.684-1.0946-3.3453-1.5304s-1.1536-0.93054-1.4766-1.4843c-0.323-0.55371-0.4845-1.2151-0.4845-1.9841 0-1.374 0.55883-2.4584 1.6765-3.2531 1.1177-0.79468 2.6506-1.192 4.5989-1.192 1.815 0 3.5889 0.36914 5.3218 1.1074l-0.90747 2.0764c-1.6919-0.69727-3.2248-1.0459-4.5989-1.0459-1.21 0-2.1226 0.1897-2.7378 0.56909-0.61524 0.3794-0.92285 0.90234-0.92285 1.5688 0 0.45117 0.11536 0.83569 0.34607 1.1536 0.23071 0.31787 0.60241 0.62036 1.1151 0.90747s1.4971 0.70239 2.9531 1.2458c1.9995 0.72803 3.3505 1.4612 4.0529 2.1995 0.7024 0.73828 1.0536 1.6663 1.0536 2.7839z" fill="#182650" fill-rule="nonzero"></path></g></g></g></svg></a></div></section>';
```

Comment:
I think there is a component called BlockcertsLogo already, can't we leverage it?

Reply:
Ah right, I didn't see that one.